src/components/index.js

```diff
@@ -24,5 +24,6 @@ require('./scene/embedded');
 require('./scene/inspector');
 require('./scene/fog');
 require('./scene/keyboard-shortcuts');
+require('./scene/post-processing');
```

Comment:
sure sounds better, and an ever shorter form could be nice, it seems too long as it is. Maybe something like deffered?